autometa/common/external/diamond.py

```diff
@@ -366,7 +397,9 @@ def main(args):
 
 if __name__ == '__main__':
     import argparse
-    parser = argparse.ArgumentParser('Retrieves blastp hits with provided input assembly')
+    parser = argparse.ArgumentParser(description="""
+    Retrieves blastp hits with provided input assembly
+    """)
```

Comment:
`"""` specify a comment block where single or double quotes specify the line. Therefore, if you wanted to have multiple lines of comments, you would need to quote each line. Using `"""` alllows me to only have to do the quoting once.

For example, I could have similarly wrapped the text

```python
>>>description="Some long string that will need"\
" to be wrapped due to its verbosity. Notice I am using the backslash to specify"\
" continuation and the string continues to wrap the next line. However, using"\
" backslashes and multiple, lines like this may be more difficult to read and may"\
" be easier to incorrectly write the continuation of the string")

>>>print(description)
'Some long string that will need to be wrapped due to its verbosity. Notice I am using the backslash to specify continuation and the string continues to wrap the next line. However, using backslashes and multiple, lines like this may be more difficult to read and may be easier to incorrectly write the continuation of the string'
```

Reply:
What would happen in case you don't put the backslash above. Would the lines still be wrapped?
If not then why use `"""` ?

Reply:
I encourage you to try the different approaches to see for yourself. 👍